rrmngmnt/package_manager.py

```diff
@@ -62,6 +63,31 @@ def exist(self, package):
         )
         return self._run_command_on_host(cmd)
 
+    def list_(self):
+        """
+        List installled packages on host
+
+        :return: installed packages
+        :rtype: list
+        :raise: NotImplementedError
+        """
+        if not self.list_command_d:
+            raise NotImplementedError(
+                "There is no 'list_' command defined."
+            )
+        cmd = self.list_command_d
+        self.logger.info(
+            "Getting all instaled packages from host %s", self.host
+        )
+        rc, out, err = self.host.executor().run_cmd(cmd)
+        if not rc:
+            return out.split('\n')
+        self.logger.error(
+            "Failed to get installed packages on host %s, rc: %s, err: %s",
+            self.host, rc, err
+        )
+        return []
```

Comment:
I prefer to raise exception with all info inside  ... how the user's program using this library is able to figure difference between error and no packages?